regression-test/suites/nereids_rules_p0/mv/partition_mv_rewrite.groovy

```diff
@@ -132,10 +132,12 @@ suite("partition_mv_rewrite") {
     l_suppkey;
     """
 
-    sql """DROP MATERIALIZED VIEW IF EXISTS mv_10086"""
-    sql """DROP TABLE IF EXISTS mv_10086"""
+
+    def mv_name = "mv_10086"
+    sql """DROP MATERIALIZED VIEW IF EXISTS ${mv_name}"""
+    sql """DROP TABLE IF EXISTS ${mv_name}"""
```

Comment:
not need run drop table

Reply:
i think this is needed since mv and table name are use same space